src/features/duplicates/l10n/messageIds.ts

```diff
@@ -14,7 +14,12 @@ export default makeMessages('feat.duplicates', {
       noValue: m('No value'),
       title: m('Data to merge'),
     },
+    findCandidateManually: m('Type to find another potential duplicate'),
```

Comment:
Minor detail, but I think it would be nice if this message would change depending on if there are one or more people in the list. The word "another" is not so suitable when there is only one person in the list: 
![bild](https://github.com/user-attachments/assets/f7dad2b4-f3e4-4ad0-904e-899cef62c207)

 I would suggest it being "Type to find a potential duplicate" for single, and "...another..." for multiple :)

Reply:
I took the easy route in 2c4a20ad87c93a8fa82ec4830f348399afc8e88b and just changed it to "Type to find a potential duplicate". 😊